paasJobs/configure_repo.sh

```diff
@@ -88,6 +102,7 @@ fi
 EOF
 
 install_app_config
+ensure_gamechangerml_is_installed
 install_topic_models
```

Comment:
Does the installation of gamechangerml require the topic model? If so installing topic models should come before occur before installing gamechangerml repo. @rha930 @vctrstrm

Reply:
nope, topic model is post-install